code/lib/cli/src/generators/baseGenerator.ts

```diff
@@ -212,7 +212,6 @@ export async function baseGenerator(
     storybookConfigFolder,
     componentsDestinationPath,
     webpackCompiler,
-    installStorybookPackage,
```

Comment:
There are still 3 references in the codebase remaining. I think it is save to delete them, since it isn't used anymore. On the other hand, `code/lib/cli/src/generators/NUXT/index.ts` sets the variable to `false`. We should figure out why the CLI shouldn't be installed for Nuxt.

Reply:
Yeah, I kinda left it in place because though the feature does not make sense with the architecture, I also didn't know how to proceed. Considering this was recently added, I was hoping we'd be able to find a way to make both work well with each other.

But you're right. I'll remove the remaining bits in this PR.